src/operator/contrib/roi_align.cc

```diff
@@ -213,14 +214,21 @@ void ROIAlignForward(
 num_threads(engine::OpenMP::Get()->GetRecommendedOMPThreadCount())
```

Comment:
From our local test, the better performance will be gained if we move this OMP into `n_rois` loop, L160.
Could you help change this and test the performance with your cases? Thanks.

Reply:
Time complexity of the new case should be the same with the original one. I did the changes as you suggested and test the runtime for both cases. On my server with two E5-2683 v3, this change did bring obvious performance boost.